### DISCUSSION TITLE
Screen resolution not available on headless servers

### DISCUSSION THREAD

**Discussion by strich** [rustdesk/rustdesk#6114] [General]
### Bug Description

Recently tried to migrate away from RDP on a Windows 10 headless server and RustDesk is stuck on 1024x768 with no option to change it. 

If I setup RustDesk to support sharing the RDP session, connect via RDP, then connect via RustDesk it will capture the 2k res RDP sets the machine to.

RustDesk is currently not viable as a solution without this basic feature.

### How to Reproduce

Connect to any headless Win10 machine.

### Expected Behavior

Like RDP, TeamViewer, etc I would expect RustDesk to set the resolution of the virtual monitor to the native resolution or a selected user choice.

### Operating system(s) on local side and remote side

Windows 11 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

Can do if required.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/4359#issuecomment-1690920876

**Reply by Dark-Mater** [reply]
### **I Have the same/similar issue**
**Unable to adjust remote connection resolution, window size will not scale or adjust.**

**Description**
When connecting to a Windows 10 IoT Enterprise LTSC on a headless Intel NUC on the local network there is an issue with the window resolution being fixed at 1024x768 making it too small to use effectively. Can not adjust via the RustDesk controls or via the windows display settings.
Local OS, Linux Mint 22.1, Cinnamon  6.4.8,  Linux Kernel 6.8.0-60-generic.

**How to Reproduce**
Connect to the remote machine via RustDesk.

**Expected Behavior**
Window scales to best (optimal) resolution, adjusts automatically to the local resolution like it does when using _Remmina remote desktop client_ to connect to the same NUC.

**Operating system(s) on local side and remote side**
Linux Mint 22.1 Cinnamon > Windows 10 IoT Enterprise LTSC 21H2

**RustDesk Version(s) on local side and remote side**
1.4.0 > 1.4.0

**Reply by Dark-Mater** [reply]
**Update**
Attaching a monitor to it resolved the issue.
Looks like I need to get some
_HDMI Dummy Plug-Virtual Monitor Display Emulator Headless Adapters_ 
If I want to use RustDesk at a resolution of over 1044x768.
